var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -181,6 +182,11 @@ def configure_args(self):
             else:
                 config_args.append('--disable-parallel4')
 
+        if '+jna' in self.spec:
+            config_args.append('--enable-jna')
+        else:
+            config_args.append('--disable-jna')
```

Comment:
You can replace this with a single line:
```python
config_args += self.enable_or_disable('jna')
```

Reply:
Done.